Resources/Prototypes/_Funkystation/BloodCult/Actions/spells.yml

```
@@ -0,0 +1,178 @@
+    #entityIcon: <an "EntityUid?">  # overrides the icon
+    #sound: a SoundSpecifier. Should probably be used instead of castSound down below.
+    #iconOn:  # The sprite when selected. USE THIS.
+  - type: CultistSpell
+    abilityId: CultTwistedConstruction
+    charges: 1
+    infinite: False
+    healthCost: 0
+    rechargeTime: 0
+    invocation: Ethra p'ni dedol!
+    castSound: !type:SoundPathSpecifier
+        path: /Audio/Magic/blink.ogg
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)